app/models/manager_refresh/inventory_collection.rb

```diff
@@ -35,6 +36,7 @@ def initialize(model_class: nil, manager_ref: nil, association: nil, parent: nil
       @complete                         = complete.nil? ? true : complete
       @update_only                      = update_only.nil? ? false : update_only
       @builder_params                   = builder_params
+      @inventory_object_attributes      = inventory_object_attributes
```

Comment:
Yeah, I had that check, but it was to buggy and I did not want to dig deeper.
I think right now only ansible uses the method access?!
So I actually think we could soon completely remove the `method_missing` version